lib/user_agent/browsers/chrome.rb

```diff
@@ -2,17 +2,22 @@ class UserAgent
   module Browsers
     class Chrome < Base
       def self.extend?(agent)
-        agent.detect { |useragent|
-          %w(Chrome CriOS).include?(useragent.product)
-        }
+        agent.detect do |useragent|
+          %w[Chrome CriOS].include?(useragent.product)
+        end
       end
 
-      ChromeBrowsers = %w(
+      CHROME_BROWSERS = %w[
         Iron
-      ).freeze
+      ].freeze
+
+      def ChromeBrowsers # rubocop:disable Naming/MethodName
+        warn("#{__method__} is deprecated. Please use CHROME_BROWSERS instead")
```

Comment:
Like this but I've had my logs filled by a deprecation notice like this on a hot path. Particularly annoying when it's a dependency of a dependency or something like that and so troublesome to change.

Once we've got a final Rubocop setup I'd like to change however many we have of these to something like:

```ruby
unless @chrome_browser_warned
  warn("#{__method__} is deprecated. Please use CHROME_BROWSERS instead")
  @chrome_browser_warned = true
end
```

Reply:
addressed